src/blog/2025-06-01-oxlint-stable.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Oxlint 1.0
+outline: deep
+authors:
+  - camc314
+---
+
+<AppBlogPostHeader />
+
+After eighteen months of dedicated development and hundreds of contributions from our community, Oxlint 1.0 is here! This milestone represents our commitment to building a production-ready linter that developers can rely on.
+
+Oxlint is a Rust-powered linter for JavaScript and TypeScript that ships with 500+ built-in rules and is designed to be fast, extensible, and simple to adopt. Unlike many side projects that come and go, Oxlint has been our full-time focus—we've invested deeply in creating a tool that meets the demanding needs of modern JavaScript development.
+
+## Quick Start
+
+Getting started with Oxlint is straightforward. Run it in your codebase using your preferred package manager:
+
+::: code-group
+
+```sh [npm]
+$ npx oxlint@latest
+```
+
+```sh [pnpm]
+$ pnpm dlx oxlint@latest
+```
+
+```sh [yarn]
+$ yarn dlx oxlint@latest
+```
+
+```sh [bun]
+$ bunx oxlint@latest
+```
+
+```sh [deno]
+$ deno run npm:oxlint@latest
+```
+
+:::
+
+Oxlint is configurable via an .oxlintrc.json file. Each source file is linted with the nearest configuration, and you can use overrides to target specific glob patterns. You can also extend shared configs to keep teams consistent.
```

Comment:
Note: emphasize no need to configure first.

People tend to go after configuration before running the tool. I personally want people to start with the default rules.

So something like:

For smaller projects / new projects ... blah blah just run `oxlint`

For existing projects already using eslint ... blah blah .. eslint-migrate + eslint-plugin-oxlint